nixos/doc/manual/default.nix

```
@@ -68,12 +68,34 @@ let
       optionIdPrefix = "test-opt-";
     };
 
-  sources = lib.sourceFilesBySuffices ./. [".xml"];
+  sources = runCommand "manual-sources" {
+    inputs = lib.sourceFilesBySuffices ./. [ ".xml" ".md" ];
+    nativeBuildInputs = [ pkgs.nixos-render-docs ];
+  } ''
+    mkdir $out
+    cd $out
+    cp -r --no-preserve=all $inputs/* .
+
+    declare -a convert_args
+    for mf in $(find . -type f -regex '.*\.md$'); do
```

```suggestion
    for mf in $(find . -type f -name '*.md'); do
```

Also not a fan of the `$(find)` idiom. I'd prefer at least something like

```bash
while read -r mf; do
  ...
done < <(find . -type f -name '*.md')
```

See https://mywiki.wooledge.org/BashFAQ/001